### PR TITLE
Indicator on Port Notes tab when Port has notes

### PR DESCRIPTION
**Pull request by electrocret**
Add simple identifier for when port has notes.

Please give a short description what your pull request is for

DO NOT DELETE THE UNDERLYING TEXT

#### Please note

> Please read this information carefully. You can run `./lnms dev:check` to check your code before submitting.

- [x] Have you followed our [code guidelines?](https://docs.librenms.org/Developing/Code-Guidelines/)
- [x] If my Pull Request does some changes/fixes/enhancements in the WebUI, I have inserted a screenshot of it.
- [x] If my Pull Request makes discovery/polling/yaml changes, I have added/updated [test data](https://docs.librenms.org/Developing/os/Test-Units/).

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

**Comment by electrocret** [comment]
image

File: includes/html/pages/device/port.inc.php [hunk 1]
```diff
@@ -79,7 +79,7 @@
 $menu_options['arp'] = 'ARP Table';
 $menu_options['fdb'] = 'FDB Table';
 $menu_options['events'] = 'Eventlog';
-$menu_options['notes'] = 'Notes';
+$menu_options['notes'] = (get_dev_attrib($device, 'port_id_notes:' . $port['port_id']) ?? '') == '' ? 'Notes' : 'Notes*';
```
[the remark this answers is not in the document]
Comment (electrocret) [reply includes/html/pages/device/port.inc.php:82]: > and in what case woulnd't port_id be set?

I think you missed a parenthesis when reading this. It's a copy-paste of https://github.com/librenms/librenms/blob/0a889ad36373b5509277d8245b06869074f53785/includes/html/pages/device/port/notes.inc.php#LL15C21-L15C21

[the remark this answers is not in the document]
Comment (electrocret) [reply includes/html/pages/device/port.inc.php]: I changed it to an object instead of array.

**Comment by ottorei** [comment]
This same behaviour should probably be implemented to the device notes as well for consistency.

**Review by ottorei — APPROVED** [review]

**Comment by electrocret** [comment]
> This same behaviour should probably be implemented to the device notes as well for consistency.

 I had the same thought when I made this, but decided against it cause it's easy enough to have the notes show up as a widget in the device overview page.

**Pull request merged**